### DISCUSSION TITLE
Do I need to manually import the parameters for NCR18650B in PyBamm?

### DISCUSSION THREAD

**Discussion by ghost** [pybamm-team/PyBaMM#2718] [Q&A] [answered]
Hi everyone,

I'm working on a PyBamm model for a battery system that uses the NCR18650B lithium-ion cell. I'd like to know if I need to import the parameters for this cell into PyBamm manually or if there's a built-in library that does this automatically.

I've searched the documentation and the PyBamm discussion forum, but I need help finding a clear answer to this question. If anyone has experience working with the NCR18650B in PyBamm, please let me know how you handled the parameter import process.

Also, where can I find the relevant values if I need to import the parameters manually? Are there any standard values I can use as a starting point, or do I need to perform experiments to determine the values myself?

Thank you in advance for your help and advice!

**Comment by brosaplanella** [comment, marked as answer]
We do not have the NCR18659B cell, but you can define the parameters into PyBaMM locally by defining a new script (similar in structure to [this one](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/parameters/lithium_ion/Chen2020.py)). The parameter values should be determined by running separate experiments, or taken from literature (if available).

**Comment by ghost** [comment]
Thank you so much for your response, what about NCR21700T, is there any similar parameters ?
